src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -103,9 +103,10 @@ def __init__(self, config):
         self.acctGroupUser = getattr(config.BossAir, 'acctGroupUser', "cmsdataops")
 
         # Build a requirement string
-        self.reqStr = ('stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) '
-                       '&& ((REQUIRED_OS=?="any") || stringListMember(GLIDEIN_REQUIRED_OS, REQUIRED_OS))'
-                       '&& (TARGET.Cpus >= RequestCpus)')
+        # See notes about this in PyCondorPlugin.  TODO(bbockelm): remove once HLT is upgraded.
+        self.reqStr = ('((REQUIRED_OS=?="any") || '
```

Comment:
Isn't it going to match against any sites? Since you removed the "DESIRED_Sites" logic from the requirements string.